### ISSUE TITLE
Uncaught Error: spawn EINVAL

### ISSUE THREAD

**Issue by Fublah-Man** [tintinweb/vscode-decompiler#43]
No errors in output, nothing happens with progress bar. Tried several different versions of Java, 1.8, 15, 17, 21. Ghidra GUI and analyze Headless both work. If more info is needed, let me know.

**Comment by fndxz** [comment]
I encountered the same problem as well, have you solved it?

**Comment by Fublah-Man** [comment]
No, but it's been a while since I tried using it. I'll attempt to get it working again within a week or so and give an update.
